src/fragments/lib/storage/flutter/gen2-banner.mdx

```diff
@@ -0,0 +1,5 @@
+<Callout warning>
+
+Gen 2 storage support will be available in the next major version of Amplify Flutter.
```

Comment:
New storage functionality available in Gen 2 (e.g. custom paths) is not yet available for Flutter.